server/src/utils/MixerConnection.ts

```diff
@@ -160,18 +163,22 @@ export class MixerGenericConnection {
 
     updateFadeToBlack = () => {
         state.faders[0].fader.forEach((channel: any, index: number) => {
-            this.updateOutLevel(index)
+            this.updateOutLevel(index, -1)
         })
     }
 
-    updateOutLevels = () => {
+    updateOutLevels = (allMixersButThis: number = -1) => {
         state.faders[0].fader.forEach((channel: any, index: number) => {
-            this.updateOutLevel(index)
+            this.updateOutLevel(index, -1)
             this.updateNextAux(index)
         })
     }
 
-    updateOutLevel = (faderIndex: number, fadeTime: number = -1) => {
+    updateOutLevel = (
+        faderIndex: number,
+        fadeTime: number,
+        allMixersButThis: number = -1

```

Comment:
Not a fan of the name `allMixersButThis`. 

First of all, the name indiciates it's a boolean, but is a number. If it needs to be a number `mixerIndexToSkip: number` or `mixerIndexesToSkip: number[]` will be more applicable.

Second of all, it's counterintuative what will happen when you are calling the method.
This snippet is from somewhere else in this PR:
```
mixerGenericConnection.updateOutLevel(
  assignedFaderIndex,
  0,
  this.mixerIndex
)
```
Looking at the snippet, and with no other knowledge of the codebase, the intuiative guess to as what would happen would be, that the outLevel is updated on the given faderIndex for the the given mixerIndex. But this is not what is happening.

Sidenote: It seems a bit weird to me that we have to specify a specific mixer when we are on a mixerConnection. I admit, I don't know the domain, but I would assume you would only have one Mixer pr MixerConnection, thus eliminating the need to specify mixers at all since `mixerConnection.updateOutLevel` will only ever update the mixer that it is connected to. (But I guess MixerConnection can connect to several Mixers?)

Reply:
Yes, allMixersButThis would be more of a SoundEngineer term, I agree on: mixerIndexToSkip: number

It needs to be called from the mixer that should be excluded, as it's the one that know it should be updated by itself.